torchvision/models/detection/roi_heads.py

```diff
@@ -591,30 +644,59 @@ def forward(self, features, proposals, image_shapes, targets=None):
 
             losses.update(loss_mask)
 
+
         if self.has_keypoint:
             keypoint_proposals = [p["boxes"] for p in result]
             if self.training:
                 # during training, only focus on positive boxes
                 num_images = len(proposals)
                 keypoint_proposals = []
                 pos_matched_idxs = []
+                assert matched_idxs is not None
                 for img_id in range(num_images):
                     pos = torch.nonzero(labels[img_id] > 0).squeeze(1)
                     keypoint_proposals.append(proposals[img_id][pos])
                     pos_matched_idxs.append(matched_idxs[img_id][pos])
+            else:
+                pos_matched_idxs = None
+
 
-            keypoint_features = self.keypoint_roi_pool(features, keypoint_proposals, image_shapes)
-            keypoint_features = self.keypoint_head(keypoint_features)
-            keypoint_logits = self.keypoint_predictor(keypoint_features)
+            # keypoint_features = self.keypoint_roi_pool(features, keypoint_proposals, image_shapes)
+            # keypoint_features = self.keypoint_head(keypoint_features)
+            # keypoint_logits = self.keypoint_predictor(keypoint_features)
+
+            if self.keypoint_roi_pool is not None:
```

Comment:
Is this still needed? Because the `maskrcnn` branch doesn't have a similar branching.